src/docs/cookbook/navigation/navigation-basics.md

```diff
@@ -2,18 +2,20 @@
 title: Navigate to a new screen and back
 description: How to navigate between routes
 prev:
-  title: Animating a Widget across screens
+  title: Animating a widget across screens
   path: /docs/cookbook/navigation/hero-animations
 next:
   title: Navigate with named routes
   path: /docs/cookbook/navigation/named-routes
 ---
 
-Most apps contain several screens for displaying different types of information.
-For example, an app might have a screen that displays products. Users can then
-tap the image of a product to get more detailed information on a new screen.
+Most apps contain several screens for displaying different types of
+information.
+For example, an app might have a screen that displays products.
+Users can then tap the image of a product to get more detailed
+information on a new screen.
```

Comment:
It sounds like they're getting information about a new screen. Um... Maybe:

When the user taps the image of a product, a new screen displays details about the product.